_posts/2017-10-30-starterkit-autumn-2017.md

```diff
@@ -1,46 +1,102 @@
 ---
 layout: post
 title: Starterkit Autumn 2017
-date: 2017-10-30
+date: 2018-02-05 13:00 +0100
```

Comment:
The date here should be that of the workshop.

Reply:
Done 👍